tokenizer.js

```diff
@@ -197,7 +199,7 @@
 				limit = options.numToSuggest || 1000,
 				list = [];
 				for (i = 0; word && i < words.length && list.length < limit; i++) {
-					if (!words[i].match(re1)) { continue; }
+					if (!words[i].match(re1) && !(options.allowFullSearch && word == options.fullSearchChar)) { continue; }
```

Comment:
Oh ok. This looks good then, just needs the conflicts resolved (it's out of sync w/ the master branch so I can't press the merge button).